### ISSUE TITLE
Question: Non-Standard Terminology

### ISSUE THREAD

**Issue by rimonim** [Reilly-ConceptsCognitionLab/SemanticDistance#7]
1. In a number of places, the documentation refers to GloVe embedding dimensions as “hyperparameters.” In my experience in machine learning ([as well as per Wikipedia](https://en.wikipedia.org/wiki/Hyperparameter_(machine_learning))), ‘hyperparameter’ refers to a value that configures a model’s learning process (rather than being learned directly). The GloVe algorithm indeed has a number of hyperparameters (e.g. learning rate, context window size, smoothing exponent), but the word here seems to be referring to the embeddings themselves, which are simply parameters. 
2. Lines 36-37 of the paper read, “the package instead indexes an internal lookup database containing publicly available lexical norms.” Is this referring again to GloVe embeddings? If so, I find both the terms 'lexical norms' and 'database' very strange here. Lexical norms are generally dictionaries of psychometric values derived from experimental data or human ratings - why not just say word embeddings? Likewise, 'database' suggests some complex structured format, whereas the data in question here is simply a data.frame.

Is there a particular reason for using this non-standard terminology here?

Not a requirement re: [openjournals/joss-reviews#10845](https://github.com/openjournals/joss-reviews/issues/10845)

**Comment by reilly-lab** [comment]
1. We corrected one instance of 'hyperparameter' (substituted parameter) in the introduction vignette and two references to hyperparameters in the R.data file.  We rebuilt the vignettes and pushed the new R script. 
2. We swapped 'embeddings' for database in the paper to avoid confusion and improve specificity. Thank you for this suggestion.
